turbo.json

```
@@ -15,6 +15,7 @@
       "dependsOn": ["^build"],
       "outputs": ["build", ".next/**"],
       "env": [
+        "LOCAL_API_KEY",
```

```suggestion
        "RESOURCES_SECRET_TOKEN",
```

`LOCAL_API_KEY` doesn't mean anything.